docs/src/Examples/Localization/MomentLocalizationExample.jsx

```diff
@@ -1,19 +1,91 @@
-import React from 'react';
+import React, { PureComponent } from 'react';
+import DatePicker from 'material-ui-pickers/DatePicker';
 import moment from 'moment';
-import 'moment/locale/fr'; // this is the important bit, you have to import the locale your'e trying to use.
+import 'moment/locale/fr';
+import 'moment/locale/ru';
 import MomentUtils from 'material-ui-pickers/utils/moment-utils';
 import MuiPickersUtilsProvider from 'material-ui-pickers/utils/MuiPickersUtilsProvider';
+import { Icon, IconButton, Menu, MenuItem } from '@material-ui/core';
 
 moment.locale('fr');
 
-const App = () => (
-  <MuiPickersUtilsProvider
-    utils={MomentUtils}
-    moment={moment}
-    locale="fr"
-  >
-      ...
-  </MuiPickersUtilsProvider>
-);
-
-export default App;
+const localeMap = {
+  en: 'en',
+  fr: 'fr',
+  ru: 'ru',
+};
+
+export default class MomentLocalizationExample extends PureComponent {
+  state = {
+    selectedDate: new Date(),
+    anchorEl: null,
+    currentLocale: 'fr',
+  }
+
+  handleDateChange = (date) => {
+    this.setState({ selectedDate: date.toDate() });
+  }
+
+  handleMenuOpen = (event) => {
+    event.stopPropagation();
+    this.setState({ anchorEl: event.currentTarget });
+  }
+
+  handleMenuClose = () => {
+    this.setState({ anchorEl: null });
+  };
+
+  selectLocale = (selectedLocale) => {
+    this.setState({
+      currentLocale: selectedLocale,
+      anchorEl: null,
+    });
+  }
+
+  render() {
+    const { selectedDate } = this.state;
+    const locale = localeMap[this.state.currentLocale];
+    moment.locale(locale);
```

Comment:
Would be better to move this out to selectLocale function. This would be more obvious